Orange/widgets/classify/owclassificationtreegraph.py

```diff
@@ -323,10 +323,19 @@ def rulew(self, i=0):
                 attr_name = parent_attr.name
                 sign = [">", "<="][is_left_child]
                 value = "%.3f" % self.tree.threshold[self.parent.i]
-            pr.append((attr_name, sign, value))
+            if (attr_name, sign) in pr:
+                old_val = pr[(attr_name, sign)]
+                if sign == ">":
+                    pr[(attr_name, sign)] = max(float(value), float(old_val))
+                elif sign == "≠":
+                    pr[(attr_name, sign)] = "{}, {}".format(old_val, value)
+                elif sign == "<=":
```

Comment:
So we use Unicode "not equal to" symbol but ASCII "less-than or equal to" construct?

:-1: for consistency.

Use `≤`?